pkg/service/handlers/v1-kube-api-authn.go

```diff
@@ -37,7 +38,6 @@ func (h *KubeAPIHandlers) v1Authenticate(w http.ResponseWriter, r *http.Request)
 		ReturnHTTPError(w, r, http.StatusBadRequest, fmt.Sprintf("%v", err))
 		return
 	}
```

Comment:
Probably worth leaving at least a debug level log line
```suggestion
	}
	
	log.Debugf("Processing v1Authenticate request for %s", accessKey)
```